<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connection Design v1.00</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="canvas_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>3</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>3</y>
      <width>785</width>
      <height>705</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(220,220,220);
padding: 3px;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_info">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>408</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(47, 94, 141);
background-color:rgb(220,220,220);
padding: 6px;
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuGeometry">
    <property name="title">
     <string>Shear Connection</string>
    </property>
    <addaction name="actionShearGusset"/>
    <addaction name="actionShearCope"/>
    <addaction name="actionShearClipAngle"/>
    <addaction name="actionShearEndPlate"/>
    <addaction name="actionDouble_Shear_Type"/>
   </widget>
   <widget class="QMenu" name="menuLoadCalculate">
    <property name="title">
     <string>Load and Calculate</string>
    </property>
    <addaction name="actionLoadings"/>
    <addaction name="actionCalculate"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionDescription"/>
    <addaction name="actionMaterialSpecs"/>
   </widget>
   <widget class="QMenu" name="menuMoment_Connection">
    <property name="title">
     <string>Moment Connection</string>
    </property>
    <addaction name="actionEnd_Plate_Type"/>
    <addaction name="actionEnd_Plate_with_Haunch"/>
    <addaction name="actionFlush_End_Plate_Type"/>
    <addaction name="actionFlush_End_Plate_with_Haunch"/>
   </widget>
   <widget class="QMenu" name="menuBracing_Connection">
    <property name="title">
     <string>Bracing Connection</string>
    </property>
    <addaction name="actionH_Shape_Verical_Bracing"/>
    <addaction name="actionH_Shape_Bracing_02"/>
    <addaction name="actionWT_Bracing"/>
    <addaction name="actionAngle_Bracing"/>
   </widget>
   <widget class="QMenu" name="menuColumn_Base">
    <property name="title">
     <string>Column Base </string>
    </property>
    <addaction name="actionPiined_Base"/>
    <addaction name="actionRigid_Base"/>
   </widget>
   <widget class="QMenu" name="menuSplice_Joint">
    <property name="title">
     <string>Splice Joint</string>
    </property>
    <addaction name="actionColumn_Splice_Joint"/>
    <addaction name="actionBeam_Splice_Joint"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSpecial_Connection">
    <property name="title">
     <string>Special Connection</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuGeometry"/>
   <addaction name="menuMoment_Connection"/>
   <addaction name="menuBracing_Connection"/>
   <addaction name="menuColumn_Base"/>
   <addaction name="menuSplice_Joint"/>
   <addaction name="menuSpecial_Connection"/>
   <addaction name="menuLoadCalculate"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionDescription"/>
   <addaction name="actionMaterialSpecs"/>
   <addaction name="separator"/>
   <addaction name="actionShearGusset"/>
   <addaction name="actionShearCope"/>
   <addaction name="actionShearClipAngle"/>
   <addaction name="actionShearEndPlate"/>
   <addaction name="actionDouble_Shear_Type"/>
   <addaction name="separator"/>
   <addaction name="actionEnd_Plate_Type"/>
   <addaction name="actionEnd_Plate_with_Haunch"/>
   <addaction name="actionFlush_End_Plate_Type"/>
   <addaction name="actionFlush_End_Plate_with_Haunch"/>
   <addaction name="separator"/>
   <addaction name="actionLoadings"/>
   <addaction name="actionCalculate"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/open_file.png</normaloff>../resources/icons/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="statusTip">
    <string>Open previously saved project file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/save_file.png</normaloff>../resources/icons/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save project input data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/save_as.png</normaloff>../resources/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="statusTip">
    <string>Save file to a different filename</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/exit.png</normaloff>../resources/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionShearGusset">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/gusset_type.png</normaloff>../resources/icons/gusset_type.png</iconset>
   </property>
   <property name="text">
    <string>Gusset Plate Type</string>
   </property>
   <property name="iconText">
    <string>Gusset Plate</string>
   </property>
   <property name="statusTip">
    <string>Input parameters for shear connection - gusset plate type</string>
   </property>
  </action>
  <action name="actionShearClipAngle">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/clip_type.png</normaloff>../resources/icons/clip_type.png</iconset>
   </property>
   <property name="text">
    <string>Clip Angle Type</string>
   </property>
   <property name="iconText">
    <string>Clip Angle</string>
   </property>
   <property name="statusTip">
    <string>Input parameters for shear connection - clip angle type</string>
   </property>
  </action>
  <action name="actionShearEndPlate">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/endplate_type.png</normaloff>../resources/icons/endplate_type.png</iconset>
   </property>
   <property name="text">
    <string>End Plate Type</string>
   </property>
   <property name="iconText">
    <string>End Plate</string>
   </property>
   <property name="statusTip">
    <string>Input parameters for shear connection - end plate type</string>
   </property>
  </action>
  <action name="actionLoadings">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/loads.png</normaloff>../resources/icons/loads.png</iconset>
   </property>
   <property name="text">
    <string>Loadings</string>
   </property>
   <property name="toolTip">
    <string>Loadings</string>
   </property>
   <property name="statusTip">
    <string>Input the applied loadings</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/calculate.png</normaloff>../resources/icons/calculate.png</iconset>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
   <property name="toolTip">
    <string>Calculate</string>
   </property>
   <property name="statusTip">
    <string>Calculate the connection capacity</string>
   </property>
  </action>
  <action name="actionDescription">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/project.png</normaloff>../resources/icons/project.png</iconset>
   </property>
   <property name="text">
    <string>Project Description</string>
   </property>
   <property name="iconText">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>Project Description</string>
   </property>
   <property name="statusTip">
    <string>Define general project information</string>
   </property>
  </action>
  <action name="actionMaterialSpecs">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/mat_specs.png</normaloff>../resources/icons/mat_specs.png</iconset>
   </property>
   <property name="text">
    <string>Material Specification</string>
   </property>
   <property name="iconText">
    <string>Material</string>
   </property>
   <property name="toolTip">
    <string>Material Specification</string>
   </property>
   <property name="statusTip">
    <string>Define material specifications</string>
   </property>
  </action>
  <action name="actionShearCope">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/notch_type.png</normaloff>../resources/icons/notch_type.png</iconset>
   </property>
   <property name="text">
    <string>Coped Type</string>
   </property>
   <property name="iconText">
    <string>Coped</string>
   </property>
   <property name="statusTip">
    <string>Input parameters for coped type connection</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPiined_Base">
   <property name="text">
    <string>Pinned Base</string>
   </property>
  </action>
  <action name="actionRigid_Base">
   <property name="text">
    <string>Fixed Base</string>
   </property>
  </action>
  <action name="actionDouble_Shear_Type">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/double_shear.png</normaloff>../resources/icons/double_shear.png</iconset>
   </property>
   <property name="text">
    <string>Double Shear Type</string>
   </property>
   <property name="iconText">
    <string>Double Shear</string>
   </property>
   <property name="statusTip">
    <string>Input parameters for double shear type connection</string>
   </property>
  </action>
  <action name="actionEnd_Plate_Type">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/end_plate1.png</normaloff>../resources/icons/end_plate1.png</iconset>
   </property>
   <property name="text">
    <string>End Plate Type</string>
   </property>
   <property name="iconText">
    <string>End Plate</string>
   </property>
   <property name="statusTip">
    <string>Input parameters for moment end plate type connection</string>
   </property>
  </action>
  <action name="actionEnd_Plate_with_Haunch">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/end_plate2.png</normaloff>../resources/icons/end_plate2.png</iconset>
   </property>
   <property name="text">
    <string>End Plate with Haunch</string>
   </property>
   <property name="iconText">
    <string>EP Haunch</string>
   </property>
  </action>
  <action name="actionFlush_End_Plate_Type">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/end_plate3.png</normaloff>../resources/icons/end_plate3.png</iconset>
   </property>
   <property name="text">
    <string>Flush End Plate Type</string>
   </property>
   <property name="iconText">
    <string>Flush EP</string>
   </property>
  </action>
  <action name="actionFlush_End_Plate_with_Haunch">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/end_plate4.png</normaloff>../resources/icons/end_plate4.png</iconset>
   </property>
   <property name="text">
    <string>Flush End Plate with Haunch</string>
   </property>
   <property name="iconText">
    <string>FEP Haunch</string>
   </property>
  </action>
  <action name="actionH_Shape_Verical_Bracing">
   <property name="text">
    <string>H-Shape Bracing 01</string>
   </property>
  </action>
  <action name="actionWT_Bracing">
   <property name="text">
    <string>WT Bracing</string>
   </property>
  </action>
  <action name="actionAngle_Bracing">
   <property name="text">
    <string>Angle Bracing</string>
   </property>
  </action>
  <action name="actionH_Shape_Bracing_02">
   <property name="text">
    <string>H-Shape Bracing 02</string>
   </property>
  </action>
  <action name="actionColumn_Splice_Joint">
   <property name="text">
    <string>Column Splice Joint</string>
   </property>
  </action>
  <action name="actionBeam_Splice_Joint">
   <property name="text">
    <string>Beam Splice Joint</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
